presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -243,12 +244,14 @@ protected void assertQueryFails(Session session, @Language("SQL") String sql, @L
 
     protected void assertQueryReturnsEmptyResult(@Language("SQL") String sql)
     {
-        QueryAssertions.assertQueryReturnsEmptyResult(queryRunner, getSession(), sql);
```

Comment:
don't you need to deprecate the `assertQueryReturnsEmptyResult` method?

Reply:
This doesn't bring much benefit, until someone actually updates all the call sites. 
I didn't want to do that in this PR.